### ISSUE TITLE
discussion about making it responsive

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/calendar#49]
https://github.com/GroceriStar/react-only-intern-23/issues/18

**Comment by sanchit94** [comment]
I think we are using bootstrap already in this project, so making it responsive should be less of a problem.

**Comment by atherdon** [comment]
i agree with you, but not sure that we'll make this: ***get our current work and layout and just redo it, by applying bootstrap styles***
maybe, just maybe - we'll switch into some different, but easier and better way, when our responsive layout will be solved by default.
